openmc/mesh.py

```diff
@@ -727,10 +735,10 @@ def from_rect_lattice(cls, lattice, division=1, mesh_id=None, name=''):
     @classmethod
     def from_domain(
         cls,
-        domain,
-        dimension=(10, 10, 10),
-        mesh_id=None,
-        name=''
+        domain: typing.Union['openmc.Cell', 'openmc.Region', 'openmc.Universe', 'openmc.Geometry'],
+        dimension: typing.Tuple[int] = (10, 10, 10),
```

Comment:
We allow other iterables (lists, e.g.) so this should probably be 
```suggestion
        dimension: typing.Sequence[int] = (10, 10, 10),
```

Reply:
nice, I've not used Sequence before. Thanks for the new info